docs/src/pages/system/the-sx-prop/the-sx-prop.md

```diff
@@ -0,0 +1,177 @@
+```
+
+The `borderRadius` properties multiples the value it receives by the `theme.shape.borderRadius` value (the default for the value is `4px`).
+
+```jsx
+<Box sx={{ borderRadius: 2 }} /> // equivalent to borderRadius: theme => 2 * theme.shape.borderRadius
+```
+
+_Head to the [borders page](/system/borders) for more examples._
+
+## Display
+
+The `displayPrint` property allows you to specify CSS `display` value, that will be applied only for printing.
+
+```jsx
+<Box sx={{ displayPrint: 'none' }} /> // equivalent to '@media print': { display: 'none' }
+```
+
+_Head to the [display page](/system/display) for more examples._
+
+## Grid
+
+The grid CSS properties `gap`, `rowGap` and `columnGap` multiply the values they receive by the `theme.spacing` value (the default for the value is `8px`).
+
+```jsx
+<Box sx={{ gap: 2 }} /> // equivalent to gap: theme => theme.spacing(2)
+```
+
+_Head to the [grid page](/system/grid) for more examples._
+
+## Palette
+
+The `color` and `backgroundColor` properties can receive a string, which represents the path in the `theme.palette`.
+
+```jsx
+<Box sx={{ color: 'primary.main' }} /> // equivalent to color: theme => theme.palette.primary.main
+```
+
+The `backgroundColor` property is also available trough its alias `bgcolor`.
+
+```jsx
+<Box sx={{ bgcolor: 'primary.main' }} /> // equivalent to backgroundColor: theme => theme.palette.primary.main
+```
+
+_Head to the [palette page](/system/palette) for more examples._
+
+## Positions
+
+The `zIndex` property maps its value to the `theme.zIndex` value.
+
+```jsx
+<Box sx={{ zIndex: 'tooltip' }} /> // equivalent to backgroundColor: theme => theme.zIndex.tooltip
+```
+
+_Head to the [positions page](/system/positions) for more examples._
+
+## Shadows
+
+The `boxShadow` property maps its value to the `theme.shadows` value.
+
+```jsx
+<Box sx={{ boxShadow: 1 }} /> // equivalent to boxShadow: theme => theme.shadows[1]
+```
+
+_Head to the [shadows page](/system/shadows) for more examples._
+
+## Sizing
+
+The sizing properties: `width`, `height`, `minHeight`, `maxHeight`, `minWidth` and `maxWidth` are using the following custom transform function for the value:
+
+```js
+function transform(value) {
+  return value <= 1 ? `${value * 100}%` : value;
+}
+```
+
+Basically, if the value is between [0, 1] it is converted to percent, otherwise it is directly set on the CSS property.
+
+```jsx
+<Box sx={{ width: 0.5 }} /> // equivalent to width: '50%'
+<Box sx={{ width: 20 }} /> // equivalent to width: '20px'
+```
+
+_Head to the [sizing page](/system/sizing) for more examples._
+
+## Spacing
+
+The spacing properties: `margin`, `padding` and the corresponding longhand properties multiply the values they receive by the `theme.spacing` value (the default for the value is `8px`).
+
+```jsx
+<Box sx={{ margin: 2 }} /> // equivalent to margin: theme => theme.spacing(2)
```

Comment:
Does it mean that, once we merge these changes, in https://deploy-preview-26769--material-ui.netlify.app/system/spacing/#transformation, we can remove the `sx` and document the props flat :D?

Reply:
I like that we have one page as a shortcut for defining all `sx` theme props, and then separate packages with more details for each property... I don't understand whether we really need that extensive explanation as done in https://deploy-preview-26769--material-ui.netlify.app/system/spacing/#transformation tough... Will take a look on that PR

Reply:
Oh, I see, OK fair.